source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+
+For this example, we need to filter out samples with fewer sequences than our rarefaction depth.
+
+.. command-block::
+
+   qiime feature-table filter-samples \
+     --i-table ./deblur_table.qza \
+     --p-min-frequency 1000 \
+     --o-filtered-table ./table_1k.qza
+
+Now, let’s use the filtered table to build an interactive barplot of the taxonomy in the sample.
+
+.. command-block::
+
+   qiime taxa barplot \
+     --i-table ./table_1k.qza \
+     --i-taxonomy ./taxonomy.qza \
+     --m-metadata-file ./metadata.tsv \
+     --o-visualization ./taxa_barplot.qzv
+
+.. question::
+
+   Visualize the data at level 2 (phylum level) and sort the samples by donor, then by genotype. Can you observe a consistent difference in phylum between the donors? Does this surprising you? Why or why not?
+
+
+Differential Abundance with ANCOM
+=================================
+
+Microbiome data is inherently sparse (has a lot of zeros) and compositional (everything adds up to 1). Because of this, traditional statistical methods that you may be familiar with such as anova or t-test are not appropriate for the data and lead to a high false positive rate. ANCOM is a compositionally aware alternative that allows to test for differentially abundant features. If you’re unfamiliar with the technique, it’s worthwhile to review the `ANCOM paper`_ to better understand the method.
+
+Before we being, we're going to filter out low abundance/low prevelance ASVs. Filtering can provide better resolution and limit FDR penalty on features that are too far below the noise threshhold to be applicable to a statistical test. A feature that shows up with 10 counts may be a real feature that is present only in htat sample, may be a feature that's present in several samples but only got amplified and sequenced in one sample because PCR is a somewhat stocahastic process, or it may be noise. It's not possible to tell, so feature-based analysis may be better after filtering low abundance features. However, filtering also shifts the composition of a sample further disrupting the relationship. Here, the filtering is performed as trade off between the model, computation, and statistical 
+
+.. command-block::
+
+   qiime feature-table filter-features \
+     --i-table ./table_1k.qza \
+     --p-min-frequency 50 \
+     --p-min-samples 4 \
+     --o-filtered-table ./table_1k_abund.qza
+
+ANCOM operates on a ``FeatureTable[Composition]`` Artifact, which is based on the relative abundance of features on a per-sample basis. However, the ``FeatureTable[Composition]`` object cannot tolerate zeros (because compositional methods typically use a log-transform or a ratio and you can’t take the log or divide by zeros). To remove the zeros from our table, we add a pseudocount to the ``FeatureTable[Frequency]`` object.
```

Comment:
```suggestion
ANCOM fundamentally operates on a ``FeatureTable[Frequency]`` which is based on the frequencies of features on a per-sample basis. However, ANCOM cannot tolerate zeros (because compositional methods typically use a log-transform or a ratio and you can’t take the log or divide by zeros). To remove the zeros from our table, we add a pseudocount to the ``FeatureTable[Frequency]`` object, creating a ``FeatureTable[Composition]`` in its place.
```